### PR TITLE
[BULLET] Compound Collisions

### PR DESCRIPTION
**Pull request by Lobotuerk**

### CODE REVIEW

File: bullet/src/Base.hh [hunk 1]
```diff
@@ -67,7 +67,7 @@ struct LinkInfo
   btRigidBody* link;
   Identity model;
   math::Pose3d pose;
-  btScalar mass;
+  double mass;
```
Comment (Blast545) [inline bullet/src/Base.hh:70]: Is this change needed?

Comment (Lobotuerk) [reply bullet/src/Base.hh:70]: It fixes some warnings at compilation

**Review by Blast545 — APPROVED** [review]
LGTM
